crypto/keys/pkcs11_test.go

```diff
@@ -0,0 +1,24 @@
+// +build pkcs11
+
+// Copyright 2017 Google Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package keys
+
+import "testing"
+
+func TestPkcs11(t *testing.T) {
+	// PKCS11Config support is tested by integration/log_integration.sh (when $WITH_PKCS11 == "true").
+	t.Skip("Only integration testing is implemented for PKCS#11")
```

Comment:
Could we have a simple, happy path test? If we disable integration testing for some reason this will go without coverage.